apps/docs/src/routes/concepts/interactivity.md

```diff
@@ -4,7 +4,7 @@ title: Interactivity
 
 # Interactivity
 
-[Open the interactivity example in a Svelte REPL](https://svelte.dev/repl/c2b74dcf88ec4b4681b664e202c54274?version=3.46.2)
+[Open the interactivity example in a Svelte REPL](https://svelte.dev/repl/612bfb61f8f84e299fe5083e4767b1e1?version=3.46.2)
```

Comment:
While working on this change, I added an example like the one in this REPL to this page in order to validate my changes. I'm curious if you have thoughts about having more live examples embedded in the docs, or if you prefer just the REPL links? I was also wondering if you have a better approach to testing your changes than that.

Reply:
I think live examples are better, but a REPL is so nice because you can just dive in. It makes sense to me to use REPLs in the concepts documentation and use live examples in the rest? What do you think?

Reply:
I guess the ideal thing would be a combination of both. 🙂 But I don't think the Svelte REPL lends itself that well to embedding. CodeSandbox embeds maybe?

Reply:
You can `iframe` it, did it before. Unfortunately the slider to move the view from Input <-> Output then breaks, which makes it pretty much unusable. CodeSandbox is a good idea but also hard to maintain since they belong to someone. Is there some space for a new Tool? 😃

Reply:
Maybe 😄 Anyway, this is probably the wrong place for that discussion - I was just curious about the difference between pages and your distinction sounds reasonable enough to me!